docs/index.md

```diff
@@ -45,72 +43,122 @@ Download the latest version of the CLI for your operating system and make sure t
     event-cli.exe help
     ```
 
-### 2. Pushing data to Pulse
+## 2. Pushing data to Pulse
 
-You must use the CLI or the webhook to send information to Pulse about the following key events whenever they happen in your software delivery workflow:
+To measure the performance of your team, you must send information to Pulse about the following key events whenever they happen in the software delivery workflow of your primary application or service:
 
--  [Deployments](#deployments)
 -  [Changes](#changes)
+-  [Deployments](#deployments)
 -  [Incidents](#incidents)
 
 !!! important
-    Before setting up the integration with Pulse, make sure that you have an API key that identifies your organization and authorizes you to send data to Pulse.
+    Before setting up the integration with Pulse, make sure that you have an API key provided by Pulse to identify your organization and authorize you to send data to Pulse.
 
-#### Deployments
+### Changes
 
-**For SaaS applications,** send information to Pulse whenever you deploy to production.
+Report an event to Pulse whenever your team commits a change to a repository.
 
-**For self-hosted applications,** a better option might be to send information to Pulse whenever you have a valid artifact ready to be delivered to any user.
+Pulse uses these reports to calculate the metric [Lead time for changes](metrics.md#lead-time-for-changes).
```

Comment:
I'm linking each event with the metrics that need its information. This will also help the users understand why we're asking for the information and the value they'll be able to obtain once they set up the integration.